### PR TITLE
New package: libgme-0.6.1

### PR DESCRIPTION
**Pull request by lyzech**
Could be used in mpd and gstreamer.

### CODE REVIEW

File: srcpkgs/libgme/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+checksum="dc11bea098072d540d4d52dfb252e76fc3d3af67ee2807da48fbd8dbda3fd321"
+wrksrc="game-music-emu-${version}"
+
+# foo-devel is a subpkg
+libgme-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
```
Comment (Vaelatern) [inline srcpkgs/libgme/template]: And `vmove usr/lib/*.so`

File: srcpkgs/libgme/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+build_style=cmake
+short_desc="Video game music file emulation/playback library"
+maintainer="Pierre Allegraud <pierre.allegraud@crans.org>"
+license="LGPL-2.1"
+homepage="https://bitbucket.org/mpyne/game-music-emu/wiki/Home"
+distfiles="https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-${version}.tar.bz2"
+checksum="dc11bea098072d540d4d52dfb252e76fc3d3af67ee2807da48fbd8dbda3fd321"
+wrksrc="game-music-emu-${version}"
+
+# foo-devel is a subpkg
```
Comment (Vaelatern) [inline srcpkgs/libgme/template]: Not useful comment.

File: common/shlibs [hunk 3]
```diff
@@ -2745,3 +2745,4 @@ libkea-dns++.so.1 libkea-1.1.0_1
 libkea-util-io.so.0 libkea-1.1.0_1
 libkea-threads.so.1 libkea-1.1.0_1
 libytnef.so.0 ytnef-1.8_1
+libgme.so.0 libgme-0.6.1_1
```
Comment (Vaelatern) [inline common/shlibs]: Rebase on master so this doesn't conflict for git please

**Comment by lyzech** [comment]
Thank you for your review ! I took your advices into account and amended my commit.

**Pull request merged**